backend/app/tests/api/routes/collections/test_collection_list.py

```
@@ -0,0 +1,130 @@
+    appears in the list for the current project.
+    """
+
+    project = get_project(db, "Dalgo")
+
+    response_before = client.get(
+        f"{settings.API_V1_STR}/collections/",
+        headers=user_api_key_header,
+    )
+    assert response_before.status_code == 200
+    before_payload = response_before.json()["data"]
+    before_ids = {c["id"] for c in before_payload}
```

Ruff (0.14.3)

Local variable `before_ids` is assigned to but never used

Remove assignment to unused variable `before_ids`

(F841)